skimage/feature/blob.py

```diff
@@ -167,7 +167,7 @@ def _prune_blobs(blobs_array, overlap):
 
 def blob_dog(image, min_sigma=1, max_sigma=50, sigma_ratio=1.6, threshold=2.0,
              overlap=.5,):
-    """Finds blobs in the given grayscale image.
+    r"""Finds blobs in the given grayscale image.
```

Comment:
Oh man, this is super ugly... Is it really necessary? The docstrings render correctly in the docs. I'd rather we found a way to turn off the warnings from sphinx (or wherever they're coming from)...

Reply:
That's certainly worth something! I just saw that the official sphinx docs recommends escaping the backslashes, though. Anyway I'm gonna change it to that and merge.